doc/ref_model/chapters/chapter01.rst

```diff
@@ -51,12 +51,13 @@ The document starts from the abstract and as it progresses it increasingly gets
 Scope
 -----
 
-This **Reference Model** document focuses on identifying the abstractions, and associated concepts, that are needed to represent the cloud infrastructure. **Figure 1-1** below highlights its scope in more details.
+This **Reference Model** document focuses on identifying the abstractions, and associated concepts, that are needed to represent the cloud infrastructure. :numref:`Fig-1-1` below highlights its scope in more details.
 
-.. image:: ../figures/ch01_scope.png
-   :alt: "Figure 1-1: Scope of Reference Model"
+.. _Fig-1-1:
```

Comment:
@pgoyal01 For the reference definition, I have a doubt about the writing and the underscore, would it not be .. _Fig-1-1 ?

Reply:
@pgoyal01 Agree with the format per documentation. I am seeing the text underlined, but it seems to be a bad effect of display. In your original file, it is correct, so do not take into account my comment.